src/Serilog.Sinks.Elasticsearch/Serilog.Sinks.Elasticsearch.csproj

```diff
@@ -50,6 +50,10 @@
       <HintPath>..\..\packages\Elasticsearch.Net.2.0.4\lib\net45\Elasticsearch.Net.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Newtonsoft.Json, Version=8.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
This adds an additional dependency on newtonsoft.json (and jt looks like it is missing in the nuspec file). I m not a fan of adding these core frameworks as a reference here. The cleaner, the better. Is this really needed?